patches/@changesets+assemble-release-plan+0.3.0.patch

```diff
@@ -1,13 +0,0 @@
-diff --git a/node_modules/@changesets/assemble-release-plan/dist/assemble-release-plan.cjs.dev.js b/node_modules/@changesets/assemble-release-plan/dist/assemble-release-plan.cjs.dev.js
-index 98b84eb..3d2e5af 100644
---- a/node_modules/@changesets/assemble-release-plan/dist/assemble-release-plan.cjs.dev.js
-+++ b/node_modules/@changesets/assemble-release-plan/dist/assemble-release-plan.cjs.dev.js
-@@ -73,7 +73,7 @@ function getDependents(releases, workspaces, dependencyGraph, preInfo) {
-         versionRange
-       } = getDependencyVersionRange(dependentPkgJSON, nextRelease.name); // Firstly we check if it is a peerDependency because if it is, our dependent bump type needs to be major.
- 
--      if (depTypes.includes("peerDependencies") && nextRelease.type !== "patch" && (!releases.has(dependent) || releases.has(dependent) && releases.get(dependent).type !== "major")) {
-+      if (depTypes.includes("peerDependencies") && nextRelease.type === "major" && (!releases.has(dependent) || releases.has(dependent) && releases.get(dependent).type !== "major")) {
```

Comment:
well - this has existed for a reason :p it allows us to minor-bump XState without major-bumping dependencies its peer-dependent packages

I can bring this patch back later though

Reply:
Please do - I was unable to build without following the instructions and removing this patch.